hazelcast/src/main/java/com/hazelcast/internal/auditlog/AuditlogServiceFactory.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.auditlog;
```

Comment:
This is in the internal package, maybe I misunderstood that the SPI was public? The service is in `internal` too.

Reply:
Good point. Totally forgot about moving it into the wildness in 4.1. :) It was intentionally internal in 4.0 but it should be public now so I'll move the package out of the `*.internal`